client/app/hosting/ftp/FTP.html

```diff
@@ -154,51 +173,76 @@
 
             <!-- PRO / PERF  -->
             <div class="row" data-ng-if="ctrlHostingTabFTP.ftpInformations.hasMultiFtp">
```

Comment:
It seems that you have two nested rows and it's not be valid (ref: https://github.com/twbs/bootlint/wiki/E013).